Sources/ETCDExample/main.swift

```diff
@@ -0,0 +1,12 @@
+import ETCD
```

Comment:
I would recommend restricting this file to make it a bit more modern.

```
import ETCD

@main
struct Example {
    static func main() async throws {
         ...
     }
}
```

Doing this will also require you to move it from a `main.swift` to the `ETCDExample.swift` file